pyhf/tensor/mxnet_backend.py

```diff
@@ -108,6 +108,10 @@ def astensor(self, tensor_in, dtype='float'):
         dtype = dtypemap[dtype]
         try:
             tensor = nd.array(tensor_in, dtype=dtype)
+            try:
+                tensor.shape[0]
+            except IndexError:
```

Comment:
add an extra comment here about this exception

Reply:
I added a 1-liner. @kratsg  let me know if this should be more clear (my brain is gone at this point in the evening).